Mobile Buy SDK/Mobile Buy SDK/Models/BUYCart.m

```diff
@@ -115,4 +115,35 @@ - (NSDictionary *)jsonDictionaryForCheckout
 	return cart;
 }
 
+#pragma mark - BUYPersistence
+
++ (instancetype)instanceWithPlistDictionary:(NSDictionary *)dictionary
+{
+	if (![dictionary isKindOfClass:[NSDictionary class]]) {
```

Comment:
We would prefer not to use defensive programming techniques. Callers are responsible for providing valid data when invoking a method. If they provide invalid data, the program should crash.